docs/configure.md

```diff
@@ -236,3 +236,26 @@ region_str = "" if region_index == 0 else sep + short_region_name
 print(account_name + sep + role_name + region_str)
 ```
 If this was stored as `profile_formatter.py`, it could be used as `--profile-name-process "python profile_formatter.py"`
+
+### Change the case of profile names
+
+If you don't want to run (and maintain) a process completely outside of the
```

Comment:
Because this isn't arbitrary transformations, I don't think this needs to mention profile name process. Let's also move this section to after trim and before process